packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -50,42 +58,56 @@ abstract class Logger {
   Status startProgress(
     String message, {
     String progressId,
-    bool expectSlowOperation = false,
+    bool expectSlowOperation,
     int progressIndicatorPadding = kDefaultStatusPadding,
   });
 }
 
 class StdoutLogger extends Logger {
-
   Status _status;
 
   @override
   bool get isVerbose => false;
 
   @override
-  void printError(String message, { StackTrace stackTrace, bool emphasis = false }) {
+  void printError(
+    String message, {
+    StackTrace stackTrace,
+    bool emphasis,
+    TerminalColor color,
+  }) {
+    message ??= '';
     _status?.cancel();
     _status = null;
-    if (emphasis)
+    if (emphasis ?? false)
```

Comment:
`emphasis == true` is conceptually faster than `emphasis ?? false` since the latter is equivalent to `emphasis ~= null ? emphasis : false`.

Reply:
Good point, thanks.  Fixed, and below.